README.md

```diff
@@ -1 +1,10 @@
 # OperandSage
+
+## Introduction
+OperandSage is a tool that could analyze, process and do statistics for add and multiply operands in neural networks. The project is based on MATLAB and it is still under development.
+
+## Prepare the Operands
+
+## Set up Parameters
+
+## Build
```

Comment:
Would be better if you could add another section of contact information to make this README more formal.

Reply:
> Would be better if you could add another section of contact information to make this README more formal.

Thanks, added.